opencensus/exporters/stats/stackdriver/stackdriver_exporter.h

```diff
@@ -15,22 +15,36 @@
 #ifndef OPENCENSUS_EXPORTERS_STATS_STACKDRIVER_STACKDRIVER_EXPORTER_H_
 #define OPENCENSUS_EXPORTERS_STATS_STACKDRIVER_STACKDRIVER_EXPORTER_H_
 
+#include <string>
+
+#include "absl/base/macros.h"
 #include "absl/strings/string_view.h"
 
 namespace opencensus {
 namespace exporters {
 namespace stats {
 
+struct StackdriverOptions {
+  // The Stackdriver project ID to use.
+  std::string project_id;
+
+  // The opencensus_task is used to uniquely identify the task in Stackdriver.
+  // The recommended format is "{LANGUAGE}-{PID}@{HOSTNAME}". If PID is not
+  // available, a random number may be used.
+  std::string opencensus_task;
```

Comment:
Please file an issue to not expose this to the user. This is an implementation detail.

Reply:
Filed https://github.com/census-instrumentation/opencensus-cpp/issues/183